tools/dockerfreeze

```diff
@@ -1,15 +1,16 @@
-#!/bin/sh
+#!/bin/false
+# This is BASH completion file expected to be sourced, do not invoke it
```

Comment:
I would propose adding a copyright here alike

# Created by .. under ... license (link/to/license) in year

Reply:
Ok

Reply:
This is all under MIT though and this all owned by TypeFox AFAIK

Reply:
So they aren't necessary